core/bessctl.cc

```diff
@@ -344,6 +344,54 @@ static Module* create_module(const std::string& name,
   return m;
 }
 
+static void collect_tc(const bess::TrafficClass* c, int wid,
```

Comment:
Nit: I believe these function names should be camel case to match the style guide (worth running it through Google cpplint).